appshell/client_handler_mac.mm

```diff
@@ -56,9 +56,12 @@
   NSWindow* window = [view window];
   std::string titleStr(title);
   NSString* str = [NSString stringWithUTF8String:titleStr.c_str()];
-  [window setTitle:str];
```

Comment:
I think you missed telling me you were in full screen mode 😄 But that is fine as I figured that out from the window that you had posted.